DriverLoader.cpp

```diff
@@ -91,12 +93,19 @@ namespace vive {
             throw CouldNotLoadEntryPoint();
         }
         factory_ = reinterpret_cast<DriverFactory>(proc);
-#elif defined(OSVR_MACOSX)
-#error                                                                         \
-    "Implementation incomplete! Load dynamic library here and retrieve entry point function here!"
-#elif defined(OSVR_LINUX)
-#error                                                                         \
-    "Implementation incomplete! Load dynamic library here and retrieve entry point function here!"
+#elif defined(OSVR_LINUX) || defined(OSVR_MACOSX)
+        impl_->driver_ = dlopen(driverFile.c_str());
+        if (!impl_->driver_) {
+            reset();
+            throw CouldNotLoadDriverModule(dlerror());
+        }
+
+        auto proc = dlsym(impl_->driver_, ENTRY_POINT_FUNCTION_NAME);
+        if (!proc) {
+            reset();
+            throw CouldNotLoadEntryPoint(dlerror());
+        }
+        factory_ = reinterpret_cast<DriverFactory>(proc);                                                                       \
```

Comment:
Why is there a backslash at the end of this line? Seems to break the build for me